Handle exception thrown for sprints without completedIssuesEstimateSum.

kc284 opened the pull request:
Quick solution to get the board running again. The completedIssuesEstimateSum is
deprecated and removed in greater versions of the jira library. The script will
have to be modified if we upgrade.

Signed-off-by: Konstantina Chremmou <konstantina.chremmou@citrix.com>

robhoes commented:
Travis says no:

>C:107,12: Invalid variable name "s" (invalid-name)

It is probably applying strict Python naming conventions.

kc284 commented:
Oh true, the default pycharm settings do not complain about this and I forgot to run the script through pylint :(

kc284 committed:
Handle exception thrown for sprints without completedIssuesEstimateSum.

Quick solution to get the board running again. The completedIssuesEstimateSum is
deprecated and removed in greater versions of the jira library. The script will
have to be modified if we upgrade.

Signed-off-by: Konstantina Chremmou <konstantina.chremmou@citrix.com>
